tools/SetupFlow/DevHome.SetupFlow/Services/WindowsPackageManager.cs

```diff
@@ -185,6 +191,26 @@ public async Task<bool> RegisterAppInstallerAsync()
         }
     }
 
+    public async Task<IList<IWinGetPackage>> GetPackagesAsync(ISet<Uri> packageUriSet)
```

Comment:
Probably could call this TryGetPackagesAsync since its not guaranteed we get any packages right since inside we're using the TryGetPackageId method.